target/start-mailserver.sh

```diff
@@ -131,6 +134,7 @@ function register_functions
 
   _register_setup_function "_setup_docker_permit"
 
+  _register_setup_function "_setup_namespaces"
```

Comment:
`_register_setup_function "_setup_dovecot_namespaces"` would be more clear.

Reply:
@matejak could you refactor this as well? I agree with @casperklein :)